README.md

```diff
@@ -307,6 +307,23 @@ messages.after-failure = ""
 # A message to print after a successful build.
 messages.after-success = ""
 
+# Add the wheel install path to the CMake prefix paths.
+search.use-install-prefix = true
+
+# Add the wheel build path to the CMake prefix paths.
+search.use-build-prefix = true
+
+# List or dict of CMake module search paths. Dict from is used to override
+# another package's entry-point definition. Populates `CMAKE_MODULE_PATH`.
+search.modules = ""
```

Comment:
I think supporting both lists and dicts over complicates it. I'd assume the values are Python modules; if they were paths you could just easily do this from CMake, and we try not to add things that can be done easily from CMake for the most part.

We could possibly replace this with a filter that allowed users to filter out entry points by name (that was my original idea), and everything else could be done from CMake, perhaps?

Reply:
Done, tentative name: `search.ignore-entry-point`